cgk_calcium_tools/filehandler.py

```diff
@@ -358,112 +503,205 @@ def run_step(self, op:str,overwrite:bool=False, verbose=False, **kws):
 
-        if self.processing_steps.index(op)==0: #if first step
-            #check if all exist
+        if self.processing_steps.index(op) == 0:  # if first step
+            # check if all exist
             for finput, _ in self.get_pair_filenames(op):
-                assert os.path.exists(finput),f'''File {finput} not exist: 
-                    Run .de_interleave() to De-interleave multiplane movies'''
+                assert os.path.exists(finput), f"""File {finput} not exist:
+                    Run .de_interleave() to De-interleave multiplane movies"""
 
-        if op.startswith('PP'):
-            print('Preprocessing movies, please wait...')
+        if op.startswith("PP"):
+            print("Preprocessing movies, please wait...")
             preprocess_step(pairlist, parameters, verbose)
-        
-        if op.startswith('BP'):
+
+        if op.startswith("BP"):
             print("Applying bandpass filter, please wait...")
             spatial_filter_step(pairlist, parameters, verbose)
-                        
-        if op.startswith('MC'):
+
+        if op.startswith("MC"):
             print("Applying motion correction. Please wait...")
-            actual_idx = self.processing_steps.index(op)
-            
-            op_prev = self.processing_steps[actual_idx - 1]
-            translation_files = self.get_results_filenames("translations.csv",
-                                                            op=op)
-            crop_rect_files = self.get_results_filenames("crop_rect.csv",
-                                                            op=op)
-
-            motion_correct_step(translation_files,crop_rect_files,parameters,
-                                pairlist,verbose)
-
-        
-        if op.startswith('TR'):
-            print('Trim movies...')
-            trim_movie(pairlist, parameters , verbose)
-                                 
-        print ('done')
-    
-    def project_movie(self, input_name:str='dff',output_name:str='maxdff',
-                    operation=None,overwrite=False, verbose=False, **kws):
-        
+
+            translation_files = self.get_results_filenames(
+                "translations.csv", op=op)
+            crop_rect_files = self.get_results_filenames(
+                "crop_rect.csv", op=op)
+
+            motion_correct_step(
+                translation_files,
+                crop_rect_files,
+                parameters,
+                pairlist,
+                verbose)
+
+        if op.startswith("TR"):
+            print("Trim movies...")
+            trim_movie(pairlist, parameters, verbose)
+
+        print("done")
+
+    def project_movie(
+        self,
+        input_name: str = "dff",
+        output_name: str = "maxdff",
+        operation=None,
+        overwrite=False,
+        verbose=False,
+        **kws,
+    ) -> None:
+        """
+        This function runs the isx.project_movie funtion, wich project movies to a single statistic image.
```

Comment:
This function applies isx.project_movie to project movies to a single statistic image.